### PR TITLE
Fix pycodestyle

### PR DESCRIPTION
**Pull request by clinssen**
This fixes the static code check failure that is completely unrelated to but suddenly cropping up in https://github.com/nest/nest-simulator/pull/2058.

Also remove some unittest assertions in favour of plain Python assertions (as recommended for unittest → pytest migration).

### CODE REVIEW

File: testsuite/pytests/test_sp/test_sp_manager.py
Comment (jougs) [inline testsuite/pytests/test_sp/test_sp_manager.py:68]:
```suggestion
                assert kernel_status['syn1'] == extract_dict_a_from_b(kernel_status['syn1'], syn_dict)
```

File: testsuite/pytests/test_sp/test_sp_manager.py
Comment (jougs) [inline testsuite/pytests/test_sp/test_sp_manager.py:150]:
```suggestion
                assert 20 == len(nest.GetConnections(neurons, neurons, syn_model))
```

File: testsuite/pytests/test_sp/test_sp_manager.py
Comment (jougs) [inline testsuite/pytests/test_sp/test_sp_manager.py:124]:
```suggestion
        assert nest.structural_plasticity_update_interval == 10000.0
```

**Review by jougs — APPROVED** [review]
LGTM minus my minor comments. Still approving, trusting that you'll like them ;-)

File: testsuite/pytests/test_sp/test_sp_manager.py
Comment (jougs) [reply testsuite/pytests/test_sp/test_sp_manager.py:123]: Pretty please? With sugar on top? :laughing:

Comment (clinssen) [reply testsuite/pytests/test_sp/test_sp_manager.py]: There can be no arguing in matters of taste. Committed!

File: testsuite/pytests/test_sp/test_sp_manager.py
Comment (jougs) [inline testsuite/pytests/test_sp/test_sp_manager.py:146]: On second thought, I think the following would be nicer to read:
```suggestion
                    assert st_neuron['SE1']['z_connected'] == 10
                    assert st_neuron['SE2']['z_connected'] == 10
```

File: testsuite/pytests/test_sp/test_sp_manager.py
Comment (jougs) [inline testsuite/pytests/test_sp/test_sp_manager.py:148]:
```suggestion
                assert len(nest.GetConnections(neurons, neurons, syn_model)) == 20
```

**Review by jougs — APPROVED** [review]
Just two additional minor suggestions

**Review by jougs — APPROVED** [review]

**Comment by jougs** [comment]
Merging without second review due to simplicity of changes.

**Pull request merged**
